site/config.toml

```diff
@@ -1,6 +1,8 @@
 # Page settings
-baseURL = ""
-relativeURLs = true
+baseURL = "https://developers.mattermost.com/"
```

Comment:
Modification of baseURL was tested using hugo server option `--baseURL=https://developers.mattermost.com`